tests/EventProjectorServiceProviderTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Tests\Spatie\EventProjector;
+
+use ReflectionClass;
+use Spatie\EventProjector\Tests\TestCase;
+use Spatie\EventProjector\EventProjectionist;
+use Spatie\EventProjector\Projectors\Projector;
+use Spatie\EventProjector\Projectors\ProjectsEvents;
+
+class EventProjectorServiceProviderTest extends TestCase
+{
+    /** @var \Spatie\EventProjector\EventProjectionist */
+    private $eventProjectionist;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->eventProjectionist = app(EventProjectionist::class);
+    }
+
+    /** @test */
+    public function projectors_can_be_registered_via_config()
+    {
+        /** @var \Illuminate\Support\Collection $projectors */
+        $projectors = $this->getProtectedProperty($this->eventProjectionist, 'projectors');
+
+        $this->assertTrue($projectors->isNotEmpty());
+
+        $this->assertEquals(DummyProjector::class, $projectors->first());
+    }
+
+    /** @test */
+    public function reactors_can_be_registered_via_config()
+    {
+        /** @var \Illuminate\Support\Collection $reactors */
+        $reactors = $this->getProtectedProperty($this->eventProjectionist, 'reactors');
```

Comment:
Get we use the getter on the projectionist instead of reaching for protected properties?